chrome/webpack.config.js

```diff
@@ -14,6 +14,14 @@ module.exports = {
   },
   plugins: [
     new CopyWebpackPlugin([
+      {
+        from: "shared/_locales/en/messages.json",
+        to: "../dist/chrome/_locales/en/messages.json"
+      },
+      {
+        from: "shared/_locales/it/messages.json",
+        to: "../dist/chrome/_locales/it/messages.json"
```

Comment:
The fact that we need to update 4 different webpack configs is starting to feel a bit lame, not sure if it's worth investing time into coming up with a single webpack config, or just dealing with it.